keps/prod-readiness/sig-release/2572.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 2572
+alpha:
+  approver: "@johnbelamaric"
```

Comment:
is this line a relic? given the full deck of folks is below https://github.com/kubernetes/enhancements/pull/2567/files#diff-fd8fe06f1b2e3822da51a793f80e229a14c36824fde63e0a5639916c9f8e6aa8R16-R24

Reply:
@dims this one is the PRR approver. I think the PRR is not actually relevant, at least not in the current form, for this KEP but the existing validation required it.